src/kubectlUtils.ts

```diff
@@ -14,7 +15,12 @@ export interface Namespace {
     readonly active: boolean;
 }
 
-async function getKubeconfig(kubectl: Kubectl): Promise<any> {
+export interface ClusterConfig {
+    readonly server: string;
+    readonly certificateAuthority: string;
+}
+
+export async function getKubeconfig(kubectl: Kubectl): Promise<any> {

```

Comment:
Prefer not to export this if it is returning `any` - can we wrap the features that callers need in more strongly typed methods?

Reply:
Yes, it's an internal helper method in this file, no need to export it.